examples/deployments/uc_functions/run.py

```diff
@@ -0,0 +1,141 @@
+"""
+# Set environment variables
+export DATABRICKS_HOST="..."
+export DATABRICKS_TOKEN="..."
+export DATABRICKS_WAREHOUSE_ID="..."
+export OPENAI_API_KEY="..."
+export MLFLOW_ENABLE_UC_FUNCTIONS=true
```

Comment:
Is it necessary to export these in the environment where `python examples/deployments/uc_functions/run.py` is executed, or just `mlflow deployments start-server \
    --config-path examples/deployments/deployments_server/openai/config.yaml --port 7000`? Can we clarify?